model/src/main/proto/deprecation.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package model;
+
+import "version.proto";
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+
+// Top-level proto used to store deprecation responses. This proto is used to store the response to
+// a dialog notifying the user of an optional update, as well as the response to a dialog notifying
+// the user of an OS deprecation.
+message DeprecationResponseDatabase {
+    // Stores the response to a dialog notifying the user that an optional update is available for
+    // download.
```

Comment:
```suggestion
    // Stores a user's response to a dialog notifying them that an optional update is available for
    // download.
```

Reply:
I have updated this and other instances where the same suggestion could be applied.